packages/app/src/server/routes/forgot-password.ts

```diff
@@ -4,6 +4,7 @@ import {
 import { ReqWithPasswordResetOrder } from '../middlewares/inject-reset-order-by-token-middleware';
 
 export const forgotPassword = (req: Request, res: Response): void => {
+  // TODO: grune - make different view to bypass update password
```

Comment:
please do not add code here for the same reason above.
create a new endpoint on the login.js file  instead